pkg/iac/terraform/state/alerts.go

```diff
@@ -0,0 +1,20 @@
+package state
+
+import "fmt"
+
+type StateReadingAlert struct {
+	key string
+	err string
+}
+
+func NewStateReadingAlert(key string, err error) *StateReadingAlert {
+	return &StateReadingAlert{key: key, err: err.Error()}
+}
+
+func (s *StateReadingAlert) Message() string {
+	return fmt.Sprintf("Your analysis will be incomplete. There was an error reading state file '%s': %s", s.key, s.err)
```

Comment:
Your analysis ~~will~~ may be incomplete. There was an error reading ~~state file~~ IaC source